src/User/ProfilePage.jsx

```diff
@@ -32,16 +34,18 @@ const ProfilePage = ({ error, isLoading, user }) => {
   const { id, username, firstName, lastName, email, occupation } = user || {};
 
   return (
+    <StyledProfile>
     <StyledCard>
-      <StyledLink className="editLink" to={`/users/edit/${id}`}>Edit</StyledLink>        
+      <StyledLink className="editLink" to={`/users/${id}/edit`}>Edit</StyledLink>        
       <StyledCardHeading className="name">
         {`Hello, ${firstName} ${lastName}!`}
       </StyledCardHeading>
       <p className="username">{`Username: ${username}`}</p>
       <p className="email">{`Email: ${email}`}</p>
       <p className="occupation">{`Occupation: ${occupation}`}</p>
-    </StyledCard>
+      </StyledCard>
```

Comment:
Could probably just format this slightly nicer for readability. Shift card in 1 tab and Styled profile back one so it's indented once per level